frontend/app/screens/goal-detail-screen/goal-detail-screen.tsx

```diff
@@ -106,8 +106,11 @@ export const GoalDetailScreen = observer(function GoalDetailScreen() {
   const navigation = useNavigation();
   const route = useRoute();
   console.log(JSON.stringify(route.params));
-  const { LTgoal, STgoals, description, id } = route.params as Goal;
-  console.log(LTgoal + " " + STgoals);
+  //const { LTgoal, STgoals, description, id } = route.params as Goal;
+  const { id } = route.params as Goal;
```

Comment:
```suggestion
  const { id } = route.params as string;
```

the id is just a string, right?

Reply:
The commented lines are removed in the next PR!